doc/rst/source/fitcircle.rst

```diff
@@ -101,6 +102,9 @@ Optional Arguments
 .. |Add_-V| unicode:: 0x20 .. just an invisible code
 .. include:: explain_-V.rst_
 
+.. |Add_-a| unicode:: 0x20 .. just an invisible code
+.. include:: explain_-a.rst_
```

Comment:
We have a file `explain_-aspatial.rst_` for **-a** option, please include this file and remove your `explain_-a.rst_`.
```suggestion
.. include:: explain_-aspatial.rst_
```